### PR TITLE
Graphql Mutation

### PR DESCRIPTION
**Pull request by jycouet**
"Just" need resolvers

### CODE REVIEW

File: src/server/graphql.spec.ts [hunk 1]
```diff
@@ -18,104 +18,50 @@ describe("test graphql", () => {
...
-        Where options for \`tasks\`
-        \\"\\"\\"
         input tasksWhere {
-          id: tasksWhereid
-          title: tasksWheretitle
-          completed: tasksWherecompleted
-          category: tasksWherecategory
+          id: tasksWhere_id
+          title: tasksWhere_title
+          completed: tasksWhere_completed
```
Comment (noam-honig) [inline src/server/graphql.spec.ts]: Do we really need a separate type for title etc... wouldn't it be netter to have one for string, one for number one for bool and work with them?

Comment (jycouet) [reply src/server/graphql.spec.ts]: I was thinking about this.
But then, you have also string nullable, non nullable,...
I don't know what would be the best yet. 👀

Comment (noam-honig) [reply src/server/graphql.spec.ts]: Then I would start with fewer types - and if we need to expand, worry about that later. Agreed?

Comment (jycouet) [reply src/server/graphql.spec.ts]: Done, looks good and will not grows endless 💪

File: README.md [hunk 2]
```diff
@@ -1,12 +1,53 @@
...
+  - [ ] To test with Auth (TODO JYC)
+- [ ] api in context to not re create it at each resolver?
+- [ ] dataloader (optimisation)
+- [ ] @Field Virtual (resolving a primitive or an Entity or an Entity Virtual)
+- [ ] @Entity Virtual
+- [ ] Souscription
+
+## Open questions
+
+- How to add SQL logs?
```
Comment (noam-honig) [inline README.md:48]: SqlDatabase.LogToConsole = true

**Pull request merged**